day5/.solutions/02-GT4Py-concepts.ipynb

```diff
@@ -16,22 +16,9 @@
   },
```

Comment:
<p>could you rephrase the shortcomings of the current implementation, I didn't understand the last two sentences and what would be the desired return if domains don't match.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ofuhrer/HPC4WC/pull/88/discussion/#comment-2168570057'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>